.gitignore

```diff
@@ -0,0 +1,5 @@
+.venv
```

Comment:
You should use the toptal gitignore I shared with you - I'm expecting consistency and improvements in areas already mentioned or treated in the past

Reply:
Noted, I can't recall you sharing a toptal gitgnore with me, I could only recall you saying an empty line should be below the .gitgnore